### PR TITLE
Update distribution.yaml

### PR DESCRIPTION
**Pull request by RoBeau**
Please add the following dependency to the rosdep database.

## Package name:

boeing_gazebo_model_attachment_plugin (humble entry)
boeing_set_joint_positions_plugin

## Package Upstream Source:

https://github.com/Boeing/gazebo_model_attachment_plugin
https://github.com/Boeing/gazebo_set_joint_positions_plugin

## Purpose of using this:

This package allows for models to be linked and un linked during runtime using a ros service
Provides 

This plugin sets a RobotModel's joint values to the latest published values on a speicifed JointState topic from ROS.

# Checks
 - [x] All packages have a declared license in the package.xml
 - [x] This repository has a LICENSE file
 - [x] This package is expected to build on the submitted rosdistro

### CODE REVIEW

File: humble/distribution.yaml
Comment (clalancette) [inline humble/distribution.yaml:715]: So the big problem with this is the modified license on the repository.  Because it is not one of the standard OSI-approved licenses, we can't say that we are allowed to redistribute the results, and we are not lawyers.

I think the only ways forward here are to change this to an OSI-approved license (i.e. plain Apache-2.0), or close this PR.

**Review by clalancette — DISMISSED** [review]

File: humble/distribution.yaml
Comment (RoBeau) [reply humble/distribution.yaml:715]: We will investigate changing this to plain Apache-2.0.

Comment (mjcarroll) [reply humble/distribution.yaml:715]: @RoBeau friendly ping

Comment (RoBeau) [reply humble/distribution.yaml:715]: Hi, we have been given verbal approval to change to the plain Apache-2.0 license. Just waiting for final sign off in the next week or so.

**Comment by sloretz** [comment]
[Holding for Noetic sync](https://discourse.ros.org/t/preparing-for-noetic-sync-2023-12-11/)

File: humble/distribution.yaml
Comment (RoBeau) [reply humble/distribution.yaml:715]: @clalancette thanks for you patients, I have updated both packages to use plain Apache-2.0.

**Review by quarkytale — APPROVED** [review]

**Pull request merged**
